scripts/qiita

```diff
@@ -130,24 +130,31 @@ def load_study(owner, title, info):
               help="The parent artifacts from which the new artifact has been "
                    "processed",
               type=click.IntRange(1))
-@click.option('--command', required=False,
-              help="The command used to process the new artifact",
-              type=click.IntRange(1))
-@click.option('--parameter_set', required=False,
-              help="The parameter set used to process the new artifact",
+@click.option('--dflt_params', required=False,
```

Comment:
Thinking more about it, it doesn't really make sense because the options would be just a bunch of numbers, if we listed these by an identifier that was guaranteed to be unique that would make more sense. Also if we are going to be doing this with this command, then we should probably do the same with the rest of the commands.